src/simulators/matrix_product_state/matrix_product_state.hpp

```diff
@@ -574,6 +604,14 @@ void State::apply_ops(const std::vector<Operations::Op> &ops,
       }
     }
     //qreg_.print(std::cout);
+    // print out bond dimensions only if they may have changed since previous print
+    if (getenv("MPS_OUTPUT_DATA") && 
```

Comment:
`getenv` is expensive in run time. Maybe better to read it once at the beginning of execution, and store it in a static variable. Here and in additional locations.

Reply:
Good comment!